src/EditorFeatures/CSharpTest/IntroduceVariable/IntroduceLocalForExpressionTests.cs

```diff
@@ -391,6 +389,160 @@ void M()
     {
         int {|Rename:v|} = 1 + 1;
     }
+}");
+        }
+
+        [Fact, WorkItem(39537, "https://github.com/dotnet/roslyn/issues/39537")]
+        public async Task IntroduceDeconstruction1()
+        {
+            await TestInRegularAndScriptAsync(
+@"
+using System;
+
+class C
+{
+    (string someString, int someInt) X() => default;
+
+    void M()
+    {
+        X()[||]
+    }
+}",
+@"
+using System;
+
+class C
+{
+    (string someString, int someInt) X() => default;
+
+    void M()
+    {
+        var (someString, someInt) = X();
```

Comment:
❔ When the user configures the use of explicit types everywhere, should this be `(string someString, int someInt) = X()`?

Reply:
yes.  good point.

Reply:
ok.  this is done.